en/docs/get-started/how-to-guides/ai-data-mapping.md

```diff
@@ -0,0 +1,487 @@
+
+10. Next, click on the **+** icon in the diagram and search for `get` in the **Mediator Palette** to add the HTTP GET operation to get the weather details using the coordinates.
+
+11. Select the connection created in Step 6, provide `/data/2.5/weather` as the **Relative Path** and click **+ Add Query Param** to add the following query parameters. Finally, click **Submit** to add the operation to the integration flow.
+
+    <table>
+        <tr>
+            <th>Query Parameter Name</th>
+            <th>Is Expression</th>
+            <th>Value</th>
+        </tr>
+        <tr>
+            <td>lat</td>
+            <td>true</td>
+            <td><code>payload[0].lat</code></td>
+        </tr>
+        <tr>
+            <td>lon</td>
+            <td>true</td>
+            <td><code>payload[0].lon</code></td>
+        </tr>
+        <tr>
+            <td>appid</td>
+            <td>true</td>
+            <td><code>vars.API_KEY</code></td>
+        </tr>
+    </table>
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-weather-request.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-weather-request.png" alt="weather http request" width="30%"></a>
+
+12. Click on the **+** icon in the diagram and search for for the **Datamapper** mediator in the **Mediator Palette**.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper.png" alt="search data mapper" width="80%"></a>
+
+13. Click **+ Add new**, provide `weatherDataMapper` as the name and click **Create** in the **Create New Datamapper** form.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper-new.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper-new.png" alt="create new mapping" width="30%"></a>
+
+14. Click **Add** to add the data mapper to the integration flow.
+
+15. Click on the name of the newly added Data Mapper mediator to open the input and output mapping window.
+
+    <a href="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper-open-src.png"><img src="{{base_path}}/assets/img/get-started/how-to-guides/ai-data-mapping/ai-data-mapping-data-mapper-open-src.png" alt="open mapping file" width="80%"></a>
+
+16. To import the input schema, click on the **Import input schema**. Since this scenario involves JSON to JSON mapping, you can choose to either import from JSON sample or a JSON schema. In this guide, we will be using the **Import from JSON** option.
```

Comment:
```suggestion
16. To import the input schema, click on the **Import input schema**. Since this scenario involves JSON to JSON mapping, you can import from either a JSON sample or a JSON schema. In this guide, we will be using the **Import from JSON** option.
```

Reply:
Fixed with 26b89fe